core/src/main/java/org/mskcc/cbio/oncokb/apiModels/CancerMatch.java

```diff
@@ -0,0 +1,76 @@
+package org.mskcc.cbio.oncokb.apiModels;
+
+import org.mskcc.cbio.oncokb.model.Alteration;
+import org.mskcc.cbio.oncokb.model.Gene;
+import org.mskcc.cbio.oncokb.model.LevelOfEvidence;
+import org.mskcc.cbio.oncokb.model.TumorType;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+
+public class CancerMatch {
```

Comment:
```suggestion
public class CancerTypeMatch {
```
Let's stick to naming `CancerType`